src/assets/styles/index.css

```diff
@@ -144,3 +144,28 @@ html[data-color-scheme="dark"] #icon-checked {
 html[data-color-scheme="light"] #icon-checked {
   --svg-icon-fill: white;
 }
+
+.explorer-pages-table {
```

Comment:
Could we think of moving this or renaming it considering the place it is defined